app/src/main/java/com/ikarabulut/DefaultRules.java

```diff
@@ -1,17 +1,5 @@
 package com.ikarabulut;
 
-public class DefaultRules implements RuleSet {
-
-
-    public boolean willBeAliveNextGeneration(int aliveNeighborsCount, boolean isAlive) {
-        if (aliveNeighborsCount < 2) {
-            return isUnderpopulated();
-        } else if (isAlive && aliveNeighborsCount == 2 || aliveNeighborsCount == 3) {
-            return isFertileLand();
-        } else {
-            return isOverpopulated();
-        }
-    }
-
+public class DefaultRules extends RuleSet {
```

Comment:
@jramnani @claudiahalip  I'm curious what your thoughts are on this. I totally get what's going on here, but seeing a file with an empty class would feel sort of odd to me, but maybe my exposure to stuff like this isn't very high 🤷 . My team would 100% duplicate the same code in 30 places, so maybe I'm just used to seeing that 😅